scripts/part_info.py

```diff
@@ -0,0 +1,51 @@
+#! /usr/bin/env python
+# -*- mode: python; coding: utf-8 -*-
+# Copyright 2017 the HERA Collaboration
+# Licensed under the 2-clause BSD license.
+
+"""
+Script to handle adding a comment to the part_info table.
+"""
+
+from __future__ import print_function
+
+from hera_mc import mc, cm_utils, part_connect
+
+
+def query_args(args):
+    """
+    Gets information from user
+    """
+    if args.hpn is None:
+        args.hpn = raw_input('HERA part number:  ')
+    if args.rev is None:
+        args.rev = raw_input('HERA part revision:  ')
+    if args.comment is None:
+        args.comment = raw_input('Comment:  ')
+    if args.library_file is None:
+        args.library_file = cm_utils._query_default('library_file', args)
+    args.date = cm_utils._query_default('date', args)
+    return args
+
+
+if __name__ == '__main__':
+    parser = mc.get_mc_argument_parser()
+    parser.add_argument('-p', '--hpn', help="HERA part number", default=None)
+    parser.add_argument('-r', '--rev', help="Revision of part", default=None)
+    parser.add_argument('-c', '--comment', help="Comment on part", default=None)
+    parser.add_argument('-l', '--library_file', help="Library filename", default=None)
+    cm_utils.add_date_time_args(parser)
+    args = parser.parse_args()
+
+    if args.hpn is None or args.rev is None or args.comment is None:
+        args = query_args(args)
+
+    # Pre-process some args
+    at_date = cm_utils._get_astropytime(args.date, args.time)
+
+    db = mc.connect_to_mc_db(args)
+    session = db.sessionmaker()
+
+    # Check for part
+    print("Adding comment for part {}:{}".format(args.hpn, args.rev))
```

Comment:
This should be phrased more generically as "Adding info for part...", no? One might want to use this command to register a `library_file`.